src/main/java/Duke.java

```diff
@@ -1,10 +1,95 @@
-public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+import java.io.IOException;
+import java.util.*;
+import java.io.File;
+
+
+
+public class Duke{
+    public static void main(String[] args) throws IOException,Exception {
+        Scanner sc = new Scanner(System.in);
+        String mesInput = "";
+        ArrayList<Task> arrTask = new ArrayList<Task>();
```

Comment:
Why not instantiate the ArrayList directly inside the TaskList constructor, since it will not be updated in the main class?